src/index.ts

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+
+const PORT = process.env.PORT || 3000;
```

Comment:
I'm not sure, but you've already defined PORT to 3000 in the .env.example file, maybe you don't need to write || 3000 here again

Reply:
> I'm not sure, but you've already defined PORT to 3000 in the .env.example file, maybe you don't need to write || 3000 here again

@yuliia-anisimova The example file I created just for instance of how should we declare environmental variables because .env file everyone stores locally